### PR TITLE
DYN-6362 Refresh Packages Tour

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

Updating the Packages tour based in the new PackageManagerSearch window.
Based in the new PackageManager I did several updates for the Packages tour.
- Then text in the Popups were updated to match the menu entry for opening the packages tour.
- The dynam_guides.json was updated to match the new names of the buttons and also the location and orientation of Popups.
- Also I had to modify the hard-coded values for Width and Height for now using Bindings so in this way we can launch the Packages tour using a specifc size (otherwise the Popups were appearing in random locations due to the fixed size in PackageManagerView).
- I've updated the GIF showing the new functionality of PackageSearch).
### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

Updating the Packages tour based in the new PackageManagerSearch window.

### Reviewers

@QilongTang @dnenov

### CODE REVIEW

**Comment by RobertGlobant20** [comment]
@QilongTang the AnalyticsTests.EventTrackingDisabled() test is failing but not sure why could be related to my changes, any idea?
image

**Comment by QilongTang** [comment]
@RobertGlobant20 If you run the test locally, does it fail?

**Comment by RobertGlobant20** [comment]
> EventTrackingDisabled()

@QilongTang I've checked locally and is passing for me, Should I run the job again?
image

**Comment by QilongTang** [comment]
> > EventTrackingDisabled()
> 
> @QilongTang I've checked locally and is passing for me, Should I run the job again? image

I would give it a try while us reviewing your changes

File: src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs [hunk 1]
```diff
@@ -371,13 +383,19 @@ internal static void ExecutePackageSearch(Step stepInfo, StepUIAutomation uiAuto
...
-            PackageManagerSearchViewModel packageManagerViewModel = packageManager.DataContext as PackageManagerSearchViewModel;
+            PackageManagerSearchViewModel packageManagerViewModel = packageManager.PackageManagerViewModel.PackageSearchViewModel;
             if (packageManagerViewModel == null)
                 return;
+            packageManager.PackageManagerViewModel.Width = PMDefaultWidth;
+            packageManager.PackageManagerViewModel.Height = PMDefaultHeight;
             packageManagerViewModel.PropertyChanged -= searchPackagesPropertyChanged.Invoke;
+
+            //Enable the DynamoView.mainGrid so the user will be able to interact with Dynamo
+            (packageManager.Owner as DynamoView).EnableEnvironment(true);
```
[the remark this answers is not in the document]
Comment (QilongTang) [reply src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs:396]: Gotcha

File: src/DynamoCoreWpf/UI/GuidedTour/dynamo_guides.json [hunk 2]
```diff
@@ -365,14 +365,14 @@
...
-					"HorizontalPopupOffset": 0
-				},
+                "HostPopupInfo": {
+                    "PopupPlacement": "right",
+                    "WindowName": "PackageManagerWindow",
+                    "DynamicHostWindow": true,
+                    "HostUIElementString": "PackageManagerWindow",
+                    "VerticalPopupOffset": 150,
+                    "HorizontalPopupOffset": 0
+                },
```
[the remark this answers is not in the document]
Comment (RobertGlobant20) [reply src/DynamoCoreWpf/UI/GuidedTour/dynamo_guides.json]: Formatting updated to the original one, see the next commit:
https://github.com/DynamoDS/Dynamo/pull/14642/commits/b6181608443648fb9a6646fea9510898231f8ece

**Review by QilongTang — APPROVED** [review]

**Pull request merged**